snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/io/SnomedImportApiTest.java

```diff
@@ -203,5 +207,52 @@ public void import11ExtensionConceptWithVersion() {
 		getComponent(branchPath, SnomedComponentType.CONCEPT, "555231000005107").statusCode(200);
 		getVersion("SNOMEDCT-NE", "2015-02-05").statusCode(200);
 	}
-
+	
+	@Test
+	public void importMoreThanOneLanguageCodeDescription() {
+		final String enDescriptionId = "41320138114";
+		final String svDescriptionId = "24688171113";
+		final String conceptIdODescription = "301795004";
+		
+		getComponent(branchPath, SnomedComponentType.DESCRIPTION, enDescriptionId).statusCode(404);
+		getComponent(branchPath, SnomedComponentType.DESCRIPTION, svDescriptionId).statusCode(404);
+		getComponent(branchPath, SnomedComponentType.CONCEPT, conceptIdODescription).statusCode(404);
+		
+		final Map<?, ?> importConfiguration = ImmutableMap.builder()
+				.put("type", Rf2ReleaseType.DELTA.name())
+				.put("branchPath",  branchPath.getPath())
+				.put("createVersions", false)
+				.build();
+		
+		importArchive("SnomedCT_Release_INT_20150131_new_concept_for_languageCode.zip");
+		getComponent(branchPath, SnomedComponentType.CONCEPT, conceptIdODescription).statusCode(200);
+		
+		importArchive("SnomedCT_Release_INT_20150201_new_description_with2_language_code.zip", importConfiguration);
```

Comment:
The description in sct2_Description_Delta-sv_INT_20150201 has a language code 'en', please fix that!